client/state/plugins/installed/actions.js

```diff
@@ -67,8 +69,57 @@ export function activatePlugin( siteId, plugin ) {
 		};
 		dispatch( { ...defaultAction, type: PLUGIN_ACTIVATE_REQUEST } );
 
+		// TODO: Remove when this flux action is completely reduxified
```

Comment:
nit: I think using `@TODO` makes it easier for certain editors to find those action items. I may be wrong.

Reply:
I agree, changed in https://github.com/Automattic/wp-calypso/pull/47711/commits/2f2a9fb306b831ef3c3bbacf6324c56d54cf7431.